packages/parrot-devtools/__tests__/app/hooks/useScenarios.spec.js

```diff
@@ -137,6 +139,15 @@ describe('useScenarios', () => {
     expect(global.fetch).toHaveBeenCalledTimes(2);
   });
 
+  it('should filter scenarios when filterValue is changed', () => {
```

Comment:
We probably need another happy path test that filters out and returns just the filtered values in the array rather than empty